testsuite/Rakefile

```diff
@@ -5,14 +5,19 @@ require 'cucumber/rake/task'
 outputfile = 'output.html'
 html_results = "--format html -o #{outputfile}"
 junit_results = '--format junit -o results_junit'
-Dir.glob(File.join(Dir.pwd, 'run_sets', '*.yml')).each do |entry|
-  namespace :cucumber do
+namespace :cucumber do
+  Dir.glob(File.join(Dir.pwd, 'run_sets', '*.yml')).each do |entry|
```

Comment:
If we can inject that "entry" somewhere in the report. It will allow us to know a bit better which subset of features we are running. That will be useful for our epic TestSuite as a pipeline